.github/workflows/main.yaml

```diff
@@ -62,7 +62,8 @@ jobs:
       - name: Run tests
         run: nox -t ${{ matrix.nox-tag }} --forcecolor -- -v --cov=onnxscript --cov-report=xml --cov-append --cov-branch -n=auto --junit-xml pytest.xml
         env:
-          CATCH_ORT_SEGFAULT: "${{ matrix.os == 'ubuntu-latest' && '1' || '0' }}"
+          CATCH_ORT_SEGFAULT: "${{ matrix.os != 'windows-latest' && '1' || '0' }}"
+          CREATE_REPRODUCTION_REPORT: "${{ matrix.os != 'windows-latest' && '1' || '0' }}"
```

Comment:
why disable for windows?

Reply:
windows is slow at spawning processes. Turns out macos is not fast either. So just ubuntu for now.

Reply:
ah I remember this slow process spawning issue lol.